### PR TITLE
new event fields

### PR DESCRIPTION
**Pull request by AdeHarrison**
# How to test?
Check out and build a local docker image from this branch
[Test as part of](https://github.com/ONSdigital/census-rm-case-processor/pull/26)

# Links
[Trello](https://trello.com/c/VhVMCJ2M/813-additional-columns-for-case-events-table-8)

# Screenshots (if appropriate):

### CODE REVIEW

**Comment by AdeHarrison** [comment]
[Additional columns for Case events table ** (8)](https://trello.com/c/VhVMCJ2M/813-additional-columns-for-case-events-table-8)

File: src/main/java/uk/gov/ons/census/caseapisvc/model/entity/Event.java [hunk 1]
```diff
@@ -28,4 +33,13 @@
   @Column
   @Enumerated(EnumType.STRING)
   private EventType eventType;
+
+  @Column private UUID caseId;
```
Comment (NickGrantX) [inline src/main/java/uk/gov/ons/census/caseapisvc/model/entity/Event.java]: We can infer caseId from the relationship between the Event and the Case, so we shouldn't have this duplicate column

Comment (AdeHarrison) [reply src/main/java/uk/gov/ons/census/caseapisvc/model/entity/Event.java]: done

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Review by LukeLoze1 — APPROVED** [review]

**Pull request merged**